GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -654,12 +690,13 @@ - (NSString *)fakeDescription {
 - (BOOL)application:(UIApplication *)application
             openURL:(NSURL *)url
             options:(NSDictionary<NSString *, id> *)options {
+  SEL methodSelector = @selector(application:openURL:options:);
   // Call the real implementation if the real App Delegate has any.
-  NSValue *openURLIMPPointer = objc_getAssociatedObject(self, &kGULOpenURLOptionsIMPKey);
+  NSValue *openURLIMPPointer =
+      [[GULAppDelegateSwizzler class] originalImplementationForSelector:methodSelector object:self];
```

Comment:
Perhaps change it to `[GULAppDelegateSwizzler originalImplementationForSelector:methodSelector object:self]` here and in other places? (To keep it consistent with rest of the codebase)

Reply:
Good catch, [class] parts are leftovers from refactoring